tru_tags.php

```diff
@@ -333,18 +333,27 @@ function tru_tags_cloud_query($atts) {
 	$filter = tru_tags_filter_sections($excludesection);
 	$filter .= ($filtersearch ? filterSearch() : '');
 
+	// Using SQL caching capabilities. Txp 4.6+
+	if (function_exists('now')) {
+		$where_posted = now('posted');
+		$where_expired = " AND (".now('expires')." <= Expires OR Expires IS NULL)";
+	} else {
+		$where_posted = "now()";
+		$where_expired = " AND (now() <= Expires OR Expires = ".NULLDATETIME.")";
```

Comment:
@makss did you see this question? (I'm not very familiar with the GitHub review process)